.github/workflows/go.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Push ${{ matrix.component }} image on repo (Master)
         uses: elgohr/Publish-Docker-Github-Action@master
         with:
-          name: liqo/${{ matrix.component }}
+          name: liqo/${{ matrix.component }}-ci
```

Comment:
May we fusion this part with the development part?

Reply:
Totally agree. As a future step it may be also possible to merge all of them and use a preliminary step to set the tag as required

Reply:
Yep, I grouped the two steps together

Reply:
Yep, I grouped the two steps